packages/app/src/background/services/credentials/__tests__/credentials.test.ts

```diff
@@ -145,49 +166,41 @@ describe("background/services/credentials", () => {
       credentialsStorage.get.mockReturnValue(credentialsStorageString);
       credentialsStorage.set.mockReturnValue(undefined);
 
-      const credentialDeleted = await verifiableCredentialsService.deleteVerifiableCredential(exampleCredentialHash);
+      await verifiableCredentialsService.deleteVerifiableCredential(exampleCredentialHash);
 
-      expect(credentialDeleted).toBe(true);
       expect(credentialsStorage.set).toBeCalledTimes(1);
       expect(credentialsStorage.set).toBeCalledWith(JSON.stringify([[...credentialsMap.entries()][1]]));
     });
 
-    test("should not delete a verifiable credential if it does not exist", async () => {
+    test("should not delete a verifiable credential if it does not exist", () => {
       const [credentialsStorage] = (SimpleStorage as jest.Mock).mock.instances as [MockStorage];
       credentialsStorage.get.mockReturnValue(credentialsStorageString);
       credentialsStorage.set.mockReturnValue(undefined);
 
-      const credentialDeleted = await verifiableCredentialsService.deleteVerifiableCredential("example hash");
-
-      expect(credentialDeleted).toBe(false);
+      expect(async () => {
+        await verifiableCredentialsService.deleteVerifiableCredential("example hash");
+      }).rejects.toThrow("Verifiable Credential does not exist.");
       expect(credentialsStorage.set).toBeCalledTimes(0);
     });
 
-    test("should not delete a verifiable credential if id is empty string", async () => {
-      const credentialDeleted = await verifiableCredentialsService.deleteVerifiableCredential("");
-
-      expect(credentialDeleted).toBe(false);
-    });
-
     test("should delete all verifiable credentials", async () => {
       const [credentialsStorage] = (SimpleStorage as jest.Mock).mock.instances as [MockStorage];
       credentialsStorage.get.mockReturnValue(credentialsStorageString);
       credentialsStorage.set.mockReturnValue(undefined);
 
-      const allCredentialsDeleted = await verifiableCredentialsService.deleteAllVerifiableCredentials();
+      await verifiableCredentialsService.deleteAllVerifiableCredentials();
 
-      expect(allCredentialsDeleted).toBe(true);
       expect(credentialsStorage.clear).toBeCalledTimes(1);
     });
 
-    test("should return false when deleting all verifiable credentials from empty storage", async () => {
+    test("should return false when deleting all verifiable credentials from empty storage", () => {
       const [credentialsStorage] = (SimpleStorage as jest.Mock).mock.instances as [MockStorage];
       credentialsStorage.get.mockReturnValue(undefined);
       credentialsStorage.set.mockReturnValue(undefined);
 
-      const allCredentialsDeleted = await verifiableCredentialsService.deleteAllVerifiableCredentials();
-
-      expect(allCredentialsDeleted).toBe(false);
+      expect(async () => {
+        await verifiableCredentialsService.deleteAllVerifiableCredentials();
+      }).rejects.toThrow("No Verifiable Credentials to delete.");
```

Comment:
This is related to all expects in this file

```suggestion
      await expect(verifiableCredentialsService.deleteAllVerifiableCredentials()).rejects.toThrow("No Verifiable Credentials to delete.");
```